src/Frontend/config/routes.js

```diff
@@ -12,18 +12,23 @@
 
 import React                                         from 'react';
 import Application                                   from '../components/app/layout/Application';
-import {IndexRoute, Router, Route, hashHistory}      from 'react-router';
-import {protectPage} from '../util/react/routerHooks';
-import {syncHistoryWithStore}                        from 'react-router-redux';
-import {Provider}                                    from 'react-redux';
+import { IndexRoute, Router, Route, hashHistory }    from 'react-router';
+import { syncHistoryWithStore }                      from 'react-router-redux';
+import { Provider }                                  from 'react-redux';
 import store                                         from './redux/store';
+import { protectApp, guardFromPortal }               from '../util/security/appProfileChecker';
+import NotFoundPage                                  from '../components/app/layout/NotFoundPage';
 
 const HelloWorld = () => {
   return (
     <h1>Hello World!</h1>
   )
 };
 
+const Protected = () => {
```

Comment:
I would write these presentational components in one line. Keeps things shorter...